framework/lib/components/createable-select-dropdown/createable-select-dropdown.tsx

```diff
@@ -97,24 +97,34 @@ export const CreatableSelectDropdown = <T extends string = string> ({
   const initialSelectedOption = useMemo(() => {
     if (!isNil(value)) {
       return standardOptions.find((option) => option.value === value) ?? null
-    } if (!isNil(defaultValue)) {
-      return standardOptions.find((option) => option.value === defaultValue) ?? null
+    }
+    if (!isNil(defaultValue)) {
+      return (
+        standardOptions.find((option) => option.value === defaultValue) ?? null
+      )
```

Comment:
don't really like this duplication.
could we maybe define a function for that, where you pass in `value`/`defaultValue`?